params.go

```diff
@@ -85,3 +87,24 @@ func ParseBrowsersParam(r *http.Request) (browsers []string, err error) {
 	sort.Strings(browsers)
 	return browsers, nil
 }
+
+// ParsePathsParam returns a set list of test paths to include, or nil if no filter is provided (and all tests should be
+// included). It parses the 'paths' parameter, split on commas, and also checks for the (repeatable) 'path' params.
```

Comment:
Supporting both is harmless, but is anything made simpler on the client side by this? Seems like more things to wonder about, like "can I provide paths multiple times too"? (No.)